src/containers/App/selectors.ts

```diff
@@ -77,13 +77,13 @@ export const makeSelectUserCan = createSelector(
 export const makeSelectUserCanAccess = createSelector(
   [makeSelectUser, makeSelectUserPermissionCodeNames],
   (user, permissions) => {
-    const is_superuser = user?.is_superuser ?? false
+    const is_superuser = user?.is_superuser
```

Comment:
I am curious to know if `Boolean(user?.is_superuser)` would work here as wel

Reply:
It wouldn't - `is_superuser` should be `undefined` when the user data is not fetched yet.